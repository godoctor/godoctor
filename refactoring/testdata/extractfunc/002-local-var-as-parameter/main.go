// <<<<< extract,9,2,10,37,printB,pass
package main

import "fmt"

func main() {
	a := 3
	b := 4
	b = 5
	fmt.Println("The variable b is:", b)
	fmt.Println("The variable a is:", a, b)
}
