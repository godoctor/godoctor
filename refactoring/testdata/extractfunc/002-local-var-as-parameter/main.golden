// <<<<< extract,9,2,10,37,printB,pass
package main

import "fmt"

func main() {
	a := 3
	b := 4
	b = printB()
	fmt.Println("The variable a is:", a, b)
}
func printB() int {
	var b int
	b = 5
	fmt.Println("The variable b is:", b)
	return b
}
